pkgs/development/beam-modules/build-erlang-mk.nix

```diff
@@ -39,9 +40,13 @@ let
     ''
     else setupHook;
 
-    buildInputs = [ erlang perl which gitMinimal wget ];
+    buildInputs = buildInputs ++ [ erlang perl which gitMinimal wget ];
     propagatedBuildInputs = beamDeps;
 
+    buildFlags = [ "SKIP_DEPS=1" ]
+      ++ lib.optional (enableDebugInfo || erlang.debugInfo) ''ERL_OPTS="$ERL_OPTS +debug_info"''
```

Comment:
I am not to much into erlang but for a release build it sounds suspicious to have debug info on.

Reply:
`enableDebugInfo` defaults to false and `erlang.debugInfo` probably does too.